tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -432,31 +436,35 @@ TEST(tfw_str_eq_cstr_pos, plain)
 		}
 	}
 
+	/* Addressing substring which is actually not a part of string will
+	 * raise an error. That is expected behaviour and we should not worried
+	 * about that
+	*/
+	TEST_LOG("Ignore next warning: \"Desired position is outside the string\". "
+	         "We are checking invalid argument value\n");
```

Comment:
Frankly, this looks ugly. Perhaps in this particular case it's best to remove the warning from `tfw_str_eq_cstr_pos()` and return -1? Then whoever is interested in that case can see if that's what happened.